app/services/bulk_upload/lettings/row_parser.rb

```diff
@@ -167,6 +168,23 @@ def log
 
 private
 
+  def validate_relevant_collection_window
+    return unless start_date
+    return unless bulk_upload.form
```

Comment:
nit, this could probably be combined to a single line

Reply:
done